apps/website/src/components/content/SubNav.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Disclosure,
+  DisclosureButton,
+  DisclosurePanel,
+} from "@headlessui/react";
+import IconChevronRight from "@/icons/IconChevronRight";
+import { classes } from "@/utils/classes";
+import IconMenu from "@/icons/IconMenu";
+import Link from "./Link";
```

Comment:
```suggestion
import { classes } from "@/utils/classes";

import IconChevronRight from "@/icons/IconChevronRight";
import IconMenu from "@/icons/IconMenu";

import Link from "./Link";
```
Nit: Import grouping spacing